src/System.Linq/ref/System.Linq.cs

```diff
@@ -177,6 +177,8 @@ public static partial class Enumerable
         public static System.Collections.Generic.Dictionary<TKey, TSource> ToDictionary<TSource, TKey>(this System.Collections.Generic.IEnumerable<TSource> source, System.Func<TSource, TKey> keySelector, System.Collections.Generic.IEqualityComparer<TKey> comparer) { throw null; }
         public static System.Collections.Generic.Dictionary<TKey, TElement> ToDictionary<TSource, TKey, TElement>(this System.Collections.Generic.IEnumerable<TSource> source, System.Func<TSource, TKey> keySelector, System.Func<TSource, TElement> elementSelector) { throw null; }
         public static System.Collections.Generic.Dictionary<TKey, TElement> ToDictionary<TSource, TKey, TElement>(this System.Collections.Generic.IEnumerable<TSource> source, System.Func<TSource, TKey> keySelector, System.Func<TSource, TElement> elementSelector, System.Collections.Generic.IEqualityComparer<TKey> comparer) { throw null; }
+        public static System.Collections.Generic.HashSet<TElement> ToHashSet<TElement>(this System.Collections.Generic.IEnumerable<TElement> source) { throw null; }
+        public static System.Collections.Generic.HashSet<TElement> ToHashSet<TElement>(this System.Collections.Generic.IEnumerable<TElement> source, System.Collections.Generic.IEqualityComparer<TElement> comparer) { throw null; }
```

Comment:
We had been surrounding new APIs with `#if netcoreapp11` in the ref assemblies and tests were separated out into  conditionally included *Foo*Tests.netcoreapp1.1.cs files, e.g. see #12226. Is that not necessary anymore?